.husky/pre-commit

```diff
@@ -5,4 +5,3 @@
 npx nx format
 npx nx affected -t test --no-watch
 npx nx affected -t lint --fix
-git add -A
```

Comment:
without this you will have uncommited changes after lint fix and will have to add them manually